app/ui/src/main/java/com/fsck/k9/account/AccountRemover.kt

```diff
@@ -0,0 +1,43 @@
+package com.fsck.k9.account
+
+import com.fsck.k9.Core
+import com.fsck.k9.Preferences
+import com.fsck.k9.controller.MessagingController
+import com.fsck.k9.mailstore.LocalStoreProvider
+import timber.log.Timber
+
+/**
+ * Removes an account and all associated data.
+ */
+class AccountRemover(
```

Comment:
I think a typical naming here would be `AccountRemoveUseCase`. Right now it would be the odd one out, but on the long run we should have at least some common prefix for individual-operation classes like this one.

Reply:
That makes the class name rather long. Do you really think a `UseCase` suffix is a worthwhile improvement? What exactly could be inferred from such a name? Is that information important enough to make it part of the name?

Reply:
I read "*UseCase" clearly as "class that performs a single thing". But if we don't have any protocol or interface, it's probably not worth it. Nvm.